modules/custom/cu_core/cu_core.module

```diff
@@ -170,6 +170,14 @@ function cu_core_page_alter(&$page) {
       unset($page['help']['system_help']);
     }
   }
+
+  // Fix 404 google analytics tracking
+  global $base_path;
+  $custom_404_page = variable_get('site_404', '');
+  $current_path = arg(0) . '/' . arg(1);
+  if ($custom_404_page === $current_path || arg(0) === 'navigation404') {
```

Comment:
what are `arg(0)` and `arg(1)`?

Reply:
Those are "magical" Drupal variables taken from the URI, e.g. "node/1/view" arg(0) = "node, arg(1) = 1, arg(2) = view. Not a great way to get those values but it is a Drupal thing.